<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEffect</class>
 <widget class="QDialog" name="DialogEffect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Effect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Effect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_effect">
     <item>
      <property name="text">
       <string>BURNING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STUNNED</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HEALTH_EFFECT</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_amount">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinner_value">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_duration">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinner_duration">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEffect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEffect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
